### ISSUE TITLE
aws_redshift_cluster: availability_zone should apply ForceNew on the resource

### ISSUE THREAD

**Issue by Victorion** [terraform-providers/terraform-provider-aws#5757]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
Terraform v0.11.8
+ provider.aws v1.34.0
```

### Affected Resource(s)

* aws_redshift_cluster

### Terraform Configuration Files

Before:
```hcl
resource "aws_redshift_cluster" "default" {
  cluster_identifier                  = "tf-redshift-cluster-01"
  availability_zone                   = "eu-west-1a"
  database_name                       = "mydb"
  master_username                     = "foo_test"
  master_password                     = "Mustbe8characters"
  node_type                           = "dc1.large"
  automated_snapshot_retention_period = 0
  allow_version_upgrade               = false
  skip_final_snapshot                 = true
}
```
After:
```hcl
resource "aws_redshift_cluster" "default" {
  cluster_identifier                  = "tf-redshift-cluster-01"
  availability_zone                   = "eu-west-1b"
  database_name                       = "mydb"
  master_username                     = "foo_test"
  master_password                     = "Mustbe8characters"
  node_type                           = "dc1.large"
  automated_snapshot_retention_period = 0
  allow_version_upgrade               = false
  skip_final_snapshot                 = true
}
```

### Expected Behavior

resource aws_redshift_cluster.default is replaced by a new one in the different availability zone

### Actual Behavior

Plan gives:
`~ update in-place`
Changes are not applied, because the resource doesn't support availability zone change in-place. 

Replacement is required.

### Steps to Reproduce

1. take the tf configuration from above.
2. Adjust availability_zone
3. `terraform apply`
4. Nothing happening

**Comment by bflad** [comment]
The fix for this has been merged into master and will release with version 1.35.0 of the AWS provider, likely later this week. 👍

**Comment by bflad** [comment]
This has been released in [version 1.35.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1350-september-06-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
